lib/presentation/localizations/app_localizations.dart

```diff
@@ -34,7 +34,6 @@ import 'package:intl/intl.dart';
 import 'package:linshare_flutter_app/l10n/messages_all.dart';
 
 class AppLocalizations {
```

Comment:
IMO, it not a good idea to reformat this file

Reply:
My bad :(